stellar-contract-env-host/src/test.rs

```diff
@@ -89,3 +89,31 @@ fn vec_as_seen_by_host() -> Result<(), ()> {
     assert_eq!(val0, val1);
     Ok(())
 }
+
+#[test]
+fn vec_front_and_back() -> Result<(), ()> {
+    let mut host = Host::default();
+    let scvec0: ScVec = ScVec(vec![ScVal::U32(1), ScVal::U32(2), ScVal::U32(3)].try_into()?);
```

Comment:
You shouldn't need the ScVec:
```suggestion
    let scvec0: ScVec = vec![ScVal::U32(1), ScVal::U32(2), ScVal::U32(3)].try_into()?;
```
Let me know if that doesn't work because the direct conversation was added last week.